docs/manifest.json

```diff
@@ -321,9 +321,26 @@
 									"path": "./admin/templates/managing-templates/change-management.md"
 								},
 								{
-									"title": "Devcontainers",
-									"description": "Learn about using devcontainers in templates",
-									"path": "./admin/templates/managing-templates/devcontainers.md"
+									"title": "Development containers",
```

Comment:
What is the rational in this change? IMO most people will be searching "Dev Containers" or "Devcontainers" as it is a familiar spec. I don't think there is any value in unfolding the term

- https://www.gitpod.io/docs/flex/configuration/devcontainer
- https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers